src/main/java/core/basesyntax/serviceimpl/ShopServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ShopService;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.operation.OperationHandler;
+import java.util.List;
+
+public class ShopServiceImpl implements ShopService {
+    private OperationStrategy operationStrategy;
+
+    public ShopServiceImpl(OperationStrategy operationStrategy) {
+        this.operationStrategy = operationStrategy;
+    }
+
+    @Override
```

Comment:
The method implementation should not begin with an empty line. Please remove the empty line before the method declaration.